lib/livebook/runtime/evaluator/doctests.ex

```diff
@@ -141,7 +155,15 @@ defmodule Livebook.Runtime.Evaluator.Doctests do
         use ExUnit.Case, register: false
 
         for module <- modules do
-          doctest module
+          {:docs_v1, _, _, _, _, _, member_docs} = Code.fetch_docs(module)
```

Comment:
Don't we call `fetch_docs` some point earlier to check if we should generate doctests or not? If so, it would be nice to avoid calling it multiple times. May be we should pass `%{module: module, except: ...}` to this module?

Reply:
Good call, updated :)